src/types/device.d.ts

```diff
@@ -1,8 +1,11 @@
 export interface LxdDiskDevice {
   name?: string;
-  path: string;
+  path?: string;
   pool: string;
   size?: string;
+  source?: string;
+  "limits.read"?: string;
+  "limits.write"?: string;
```

Comment:
Maybe we can have:
```suggestion
  limits?: {
    read?: string;
    write?: string;
  };
```
?

Reply:
No, we can't change it, because this is what LXD expects in the payload and how it is being encoded in JSON.